testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -780,7 +780,7 @@ class TestPBCFlag(object):
     @dec.skipif(parser_not_found('TRZ'),
                 'TRZ parser not available. Are you using python 3?')
     def setUp(self):
-        self.prec = 3
+        self.prec = 2
```

Comment:
Well ideally it should pass with a minimal number of failing tests. If we want no failing tests and the same precision we need to go to numpy 1.10.x

In the end I think we like to fix all the failures for our minimal supported version of numpy